### ISSUE TITLE
Raw SDL is not working

### ISSUE THREAD

**Issue by ngagne** [magidoc-org/magidoc#333]
### Describe the bug

I'm trying to generate the docs using the example "raw" SDL method:

```
export default {
  introspection: {
    /**
     * The raw type.
     */
    type: 'raw',

    /**
     * The raw SDL content.
     */
    content: 'type Query\n{\ttest: String\n}',
  },
}
```

But when it generates the docs it only shows the welcome page and no GraphQL types are visible. However, when I use "sdl" method, that is working correctly.

### Reproduction

Use the following "introspection" setting as part of the magidoc configuration:

```
  introspection: {
    type: 'raw',
    content: 'type Query\n{\ttest: String\n}',
  },
```

### System Info

```shell
System:
  OS: macOS 13.6.1
  CPU: (12) arm664 Apple M2 Pro
  Memory: 46.45 MB / 32.00 GB
  Shell =: 5.9 - /bin/zsh
Binaries:
  Node: 18.14.2 - /usr/local/bin/node
  npm: 9.5.0 - /usr/local/bin/npm
Browsers:
  Chrome: 122.0.6261.129
  Safari: 17.1
```

### Severity

Blocking all usage of Magidoc

**Comment by pelletier197** [comment]
Hey @ngagne, thank you for raising this issue. I have figured out the problem and deployed the fix as part of 4.2.0. Try it out and let me know if you face other issues. :slightly_smiling_face:

**Comment by ngagne** [comment]
Thanks for the quick response @pelletier197, it's working great now!
